Client/Frontend/Browser/PasswordHelper.swift

```diff
@@ -171,4 +172,95 @@ class PasswordHelper: BrowserHelper {
             })
         })
     }
+
+    func handleAuthRequest(viewController: UIViewController, challenge: NSURLAuthenticationChallenge, completion: (password: Password?) -> Void) {
+        // If there have already been too many login attempts, we'll just fail.
+        if challenge.previousFailureCount > PasswordHelper.MaxAuthenticationAttempts {
+            completion(password: nil)
+            return
+        }
+
+        var credential = challenge.proposedCredential
+
+        // If we were passed an initial set of credentials from iOS, try and use them.
+        if let proposed = credential {
+            if let user = proposed.user {
+                if !user.isEmpty {
+                    if challenge.previousFailureCount == 0 {
+                        println("Use first guess")
+                        completion(password: Password(credential: credential!, protectionSpace: challenge.protectionSpace))
+                        return
+                    }
+                } else {
+                    credential = nil
+                }
+            } else {
+                credential = nil
+            }
+        }
+
+        // If we have some credentials, we'll show a prompt with them.
+        if let credential = credential {
+            println("Use guess")
```

Comment:
Remove this too (or make a debug flag so these don't go to the console by default).